corgidrp/data.py

```diff
@@ -708,11 +706,54 @@ def copy(self, copy_data = True):
 
         return new_bp
 
+class AstrometricCalibration(Image):
+    """
+    Class for astrometric calibration file. 
+    
+    Args:
+        data_or_filepath (str or np.array); either the filepath to the FITS file to read in OR the 2D image data
+        pri_hdr (astropy.io.fits.Header): the primary header (required only if raw 2D data is passed in)
+        ext_hdr (astropy.io.fits.Header): the image extension header (required only if raw 2D data is passed in)
+        
+    Attrs:
+        boresight (np.array): the [(RA, Dec)] of the center pixel in ([deg], [deg])
+        platescale (float): the platescale value in [mas/pixel]
+        northangle (float): the north angle value in [deg]
+
+    """
+    def __init__(self, data_or_filepath, pri_hdr=None, ext_hdr=None):
```

Comment:
It should take another argument `input_dataset` that's the `input_dataset` that gets fed into `astrometric_calibration()`. We want to reference it to record all the files that were used to make this astrometric calibration file, for future documentation reference. (see other calibration classes like `Dark` and `NonLinearityCalibration`